data.sql

```diff
@@ -1,5 +1,58 @@
 /* Populate database with sample data. */
+INSERT INTO
+    animals (name)
+VALUES
+    ('Luna');
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO
+    animals (name)
+VALUES
+    ('Daisy');
+
+INSERT INTO
+    animals (name)
+VALUES
+    ('Charlie');
+
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Agumon', '2020-02-03', 0, TRUE, 10.23)
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Gabumon', '2018-11-15', 2, TRUE, 8);
+
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Pikachu', '2021-01-07', 1, FALSE, 15.04);
+
+INSERT INTO
+    animals (
+        name,
+        date_of_birth,
+        escape_attempts,
+        neutered,
+        weight_kg
+    )
+VALUES
+    ('Pikachu', '2017-05-12', 1, FALSE, 15.04);
```

Comment:
- [ ] You have a repetition of data passed here. kindly check the data provided and make sure its exactly same as you'd be making query on them which will affect your output if not inserted properly 👍